src/app/containers/header/header.component.ts

```diff
@@ -63,12 +58,14 @@ export class HeaderComponent implements OnInit {
     }
 
   updateNavItems(url): void {
-    this.navItems = this.navItems.map(item => {
-      return {
-        ...item,
-        active: item['href'] === url
-      };
-    });
+    if (this.navItems) {
+      this.navItems = this.navItems.map(item => {
+        return {
+          ...item,
+          active: item['href'] === url
```

Comment:
Why is this not `item.href` instead of `item['href']`?

Reply:
Good point. I have now changed it to NavItem.